docs/quick_start.md

```diff
@@ -65,11 +65,11 @@ module Sublayer
 
       def prompt
         <<-PROMPT
-          You are an expert programmer in \#{@technologies.join(", ")}.
+          You are an expert programmer in \\#{@technologies.join(", ")}.
```

Comment:
this is the second time i've seen these no-opt like changes. 
I'm guessing some white spacing nonsense. 
it's not a huge deal for v1, just a minor annoyance.